process/smartContract/process.go

```diff
@@ -1246,24 +1263,50 @@ func (sc *scProcessor) processSimpleSCR(
 	scResult *smartContractResult.SmartContractResult,
 	dstAcc state.UserAccountHandler,
 ) error {
-	outputAccount := &vmcommon.OutputAccount{
-		Code:         scResult.Code,
-		CodeMetadata: scResult.CodeMetadata,
-		Address:      scResult.RcvAddr,
+	if scResult.Value.Cmp(zero) <= 0 {
```

Comment:
Can we reach this condition? Or is an error actually raised sooner?

Reply:
scResult value can be 0 multiple times